src/qt/forms/overviewpage.ui

```diff
@@ -576,19 +560,6 @@
        </item>
       </layout>
      </item>
-     <item>
-      <spacer name="horizontalSpacerCenter">
```

Comment:
>Removed stretches for Overview Page from the sides

This one is for the center part. Are you sure you want to drop it too?

Reply:
yes, we don't need to stretch **empty** space between "balance" and "transaction list". Instead, we need to stretch panels "balance" and "transaction view".